SETUP/smoketests/test_tables.sql

```diff
@@ -0,0 +1,54 @@
+USE dp_db;
+SET FOREIGN_KEY_CHECKS=0;
+
+REPLACE INTO `projects` VALUES ('*UTF-8 Practice','Lummis, Charles Fletcher','English','BEGIN','notes','html','projectID5e23a810ef693','','',1704080514,1579564017,1579395673,1708309119,'','P3.proj_avail',NULL,'20200118BEGIN001','',417,1,0,'Folklore','beginner',0,'','',15,6,NULL,'TESTING','BEGIN','BEGIN','',0,'','');
+
+REPLACE INTO `users` VALUES ('userID1234567890abc', 'Adam Adminston', 'admin', 'a@example.com', 1234567890, 1234567890, 1234567890, 1, '', '', '', 10, 0, 'project_gutenberg', 1, 110, '', 0, 'admin_key');
+REPLACE INTO `users` VALUES ('userID460b20a8a8a71','BKeir','teststeel','teststeel@localhost',1175134376,1175134406,1175134457,1,'','','',10,0,'project_gutenberg',110,119,'en_US',0,NULL);
+
+REPLACE INTO `user_profiles` VALUES (110,105,'default',4,1,1,0,0,1,5,6,'',50,40,65,1,0,5,6,'',35,20,70,1,0);
+
+REPLACE INTO `usersettings` SET username='admin', setting='sitemanager', value='yes';
+
+REPLACE INTO `user_teams` VALUES (44,'%','','http://webpage.org','wickedcoder256',1,1532715008,34,'avatar_605f6ae3e7386.png','icon_5f9490f9bf32b.png',509,151);
+
+DROP TABLE IF EXISTS `projectID5e23a810ef693`;
+CREATE TABLE `projectID5e23a810ef693` (
```

Comment:
This could become a pain to maintain as we make changes to the project table. Can we not use the Project class to create the table for us?

Reply:
I waffled on this. It seemed like the path of least resistance to just set up the database directly, even if it's brittle. Let me see if I can do it in code for at least some of the cases.

Reply:
I'll see what I can do!